src/main/java/uk/gov/hmcts/reform/cmc/model/FormaliseOption.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.cmc.model;
+
```

Comment:
Is this class related to CIV-11542. cannot see any usage this is PR?

Reply:
merge problem fixed